### PR TITLE
Real Money damage boost

### PR DESCRIPTION
**Pull request by kiaen1**
I added the real money damage boost to the calcs, as a way for people to judge if it's worth it for them.

### CODE REVIEW

File: state.pyx [hunk 1]
```diff
@@ -113,6 +114,8 @@ cdef class State:
         self.resources_per_second = state.resources_per_second
         self.ad_boost = state.ad_boost
         self.gem_level = state.gem_level
+        self.boost_level = state.boost_level
+        self.boost_level = state.boost_level
```
Comment (tadpoleloop) [inline state.pyx:118]: Repeated Lines

**Review by tadpoleloop — CHANGES_REQUESTED** [review]
Besides the small nit about a repeated line, looks good to me.

File: state.pyx [hunk 2]
```diff
@@ -90,6 +90,7 @@ cdef class State:
         self.resources_per_second = [0, 0, 0, 0, 0]
         self.ad_boost = 1
         self.gem_level = 0
+        self.boost_level = 1
```
Comment (tadpoleloop) [inline state.pyx:93]: default boost_level should be 0

**Comment by kiaen1** [comment]
Alright, it's been pushed.

**Review by tadpoleloop — APPROVED** [review]

**Pull request merged**